### DISCUSSION TITLE
How to translate?

### DISCUSSION THREAD

**Discussion by yousaf465** [wlcrs/huawei_solar#559] [General]
I want to add Urdu translation for this integration, how do I do it?

**Comment by wlcrs** [comment]
Translations are located in the folder [translations](https://github.com/wlcrs/huawei_solar/tree/main/translations). Create a copy of 'en.json' named 'ur.json' and translate all the strings. If you're unsure about what must be translated and what must be left alone in the file, then take a look at the other translations to find out.

I'm looking forward to a PR with your translations!

**Comment by yousaf465** [comment]
I created initial translations. If it's correct, I will do all the strings.
